Wikipedia/Code/WMFSettingsViewController.m

```diff
@@ -591,13 +582,6 @@ - (WMFSettingsTableViewSection *)section_2 {
     NSMutableArray *items = [NSMutableArray arrayWithArray:commonItems];
     [items addObject:[WMFSettingsMenuItem itemForType:WMFSettingsMenuItemType_ExploreFeed]];
 
-    if ([[WMFYearInReviewDataController dataControllerForObjectiveC] shouldShowYearInReviewSettingsItemWithCountryCode:NSLocale.currentLocale.countryCode primaryAppLanguageCode:self.dataStore.languageLinkController.appLanguage.languageCode]) {
-        [items addObject:[WMFSettingsMenuItem itemForType:WMFSettingsMenuItemType_YearInReview]];
-        [[NSUserDefaults standardUserDefaults] wmf_setShowYirSettingToggle:YES];
-    } else {
-        [[NSUserDefaults standardUserDefaults] wmf_setShowYirSettingToggle:NO];
-    }
-
```

Comment:
Instead of removing this (which will be needed in the future, forget we're redoing this VC), it should be tied to the other logic for displaying the entry points.